src/sentry/tasks/reports.py

```diff
@@ -773,42 +785,46 @@ def get_legend_data(report):
             "dropped_transactions": dropped_transactions,
         }
 
-    # Find the reports with the most total events. (The number of reports to
-    # keep is the same as the number of colors available to use in the legend.)
-    instances = map(
-        operator.itemgetter(0),
-        sorted(
+    # Find the reports with the most total events. Note that reports are keyed
+    # on project, so this returns the list of Projects, which map to reports
+    all_projects = [
+        v[0]
+        for v in sorted(
             reports.items(),
-            key=lambda instance__report: sum(
-                sum(values) for timestamp, values in instance__report[1][0]
```

Comment:
Avoided using an index lookup on the named tuple `Report` object. `series` is the first field of the `Report` tuple. So get that.